
class MarryTests(object):
    def test_run_marry(self):
        # import mal9001.marry
        raise Exception("this is a test failure")
